#ERP42 Message of ARITV_ver1.0
#2020.05.11

bool s_value                 #S
bool t_value                 #T
bool x_value                 #X

uint8 aorm                   #Auto or Manual
bool e_stop                  #Emergency STOP

uint8 gear                   #GEAR

uint8 speed0                 #SPEED0
uint8 speed1                 #SPEED1
uint8 steer0                 #STEER0
uint8 steer1                 #STEER1
uint8 brake                  #BRAKE

uint8 enc0                   #Encoder counting0
uint8 enc1                   #Encoder counting1
uint8 enc2                   #Encoder counting2
uint8 enc3                   #Encoder counting3

uint8 alive                  #Increasing each one step

bool etx0                    #ETX0
bool etx1                    #ETX1